webpack/components/extensions/RegistrationCommands/index.js

```diff
@@ -73,7 +73,7 @@ export const RegistrationActivationKeys = ({
   isLoading,
 }) => {
   useEffect(() => {
-    onChange({ activationKeys: [] });
+    onChange({ activationKeys: undefined });
```

Comment:
`PropTypes.oneOfType([PropTypes.undefined, PropTypes.array])`

Reply:
imho it's not a best practice to pass the value as `undefined`
if we are doing this in order to hide/display proper error on the field, e.g:
hide the error when the user opens the page,
and show it upon interaction with the field,

I propose to add a `const [ wasInteracted, setInteraction ] = useState(false)`
and then use the `wasInteracted` in the right places.

This should give us more control over what we are trying to achieve.

Reply:
I'm not against it, it makes sense, just didn't know that it's a bad practice.

@jeremylenz @MariaAga How do you handle touched / untouched form fields in React components?

Reply:
I used `onBlur={() => setWasFocus(true)`  here: https://github.com/theforeman/foreman_remote_execution/pull/784